pandoc.cabal

```diff
@@ -182,6 +182,8 @@ extra-source-files:
                  INSTALL.md, AUTHORS.md, README.md,
                  CONTRIBUTING.md, BUGS, changelog,
                  man/pandoc.1
+                 -- Makefile
+                 Makefile
                  -- cabal and stack build plans
```

Comment:
The argument against adding this is that the Makefile isn't necessary to build pandoc.
It's really just a convenience for developers, like some of the things in `tools/`.